pizzeria-frontend/src/Menu.tsx

```diff
@@ -22,16 +24,27 @@ interface IFilterItem {
 
 const Menu = () => {
   const [selectedFilter, setSelectedFilter] = useState("");
-  const [filteredMenu, setFilteredMenu] = useState(menu);
-
+  const [filteredMenu, setFilteredMenu] = useState<ProductType[]>([]);
+  const [menu, setMenu] = useState<ProductType[]>([]);
+  const firstRender = useRef(true);
   const handleFilterClick = (filterType: string) => {
     setSelectedFilter(filterType);
   };
-
+  useEffect(() => {
+    if(firstRender.current){
+      fetch(window.location.origin+ "/api/products/v1.0/menu")
```

Comment:
Didn't we say that you should be using your own custom fetch that tries to refresh JWT if no unauthorized?

Reply:
Auth doesnt exist on the this branch